### DISCUSSION TITLE
Add weasis to Iso in Linux

### DISCUSSION THREAD

**Discussion by marcheschi** [nroduit/Weasis#418] [Q&A]
Hi, 
Do you know if it is possible to create an ISO with weasis under Linux.
It works fine under Windows but under linux i got the "add Weasis" button unavailable:
![image](https://github.com/nroduit/Weasis/assets/3910408/94689c80-e41f-4795-93e5-99c3eb1e68bb)

is there a way to enable it?
Thank you
Paolo

**Comment by nroduit** [comment]
This feature is limited to Windows x86-64 as mentioned in the [documentation](https://nroduit.github.io/en/tutorials/dicom-export/index.html#cddvd-image). Under Linux, the variety of systems and architectures is too large to guarantee a running portable version.

**Comment by marcheschi** [comment]
I understand. But is it possible to create the same iso as the one created
under Windows. With the windows version of weasis?
Thank you.
Paolo

Il sab 27 mag 2023, 10:52 Nicolas Roduit ***@***.***> ha
scritto:

**Comment by nroduit** [comment]
No, this is not possible, as only the binaries (launcher, native libraries, and JDK) corresponding to the target architecture are available.

It could be a plugin delivered by the future server component (successor to weasis-pacs-connector).

**Comment by unsafeunaudited** [comment]
Sorry to necropost but, isnt the AppImage format (standard?) the just the same as ISO?

I've only ever compiled 1 app in recent memory and I recall simple modification of a file package.json, and just changing a word from deb to AppImage.

I am aware that Signal isnt written on java. I was wondering if somehow the same can be done with Weasis to generate an AppImage file (basically an ISO) along with generating the rest of the binary packages?

**Comment by nroduit** [comment]
Yes, but Appimage is for Linux only.

The aim is to avoid multiplying implementations depending on the system or distribution.
For reasons of security and interoperability of Weasis in a web context or with the system, it is preferable to install Weasis from an installer (on Windows it is possible to copy the binary files, but not really on other systems).
